docs/src/getting_started.md

```diff
@@ -1,61 +1,111 @@
 
 ClimaLand is provided as a Julia package, so it requires having Julia installed. Information about Julia packages is available on the [Julia website](https://julialang.org/packages/).
 
 First, download and install Julia by following the instructions at [https://julialang.org/downloads/](https://julialang.org/downloads/).
-Then, you can launch a [Julia REPL](https://docs.julialang.org/en/v1/stdlib/REPL/) and install the
-ClimaLand.jl package by running:
+Then, you can launch a [Julia REPL](https://docs.julialang.org/en/v1/stdlib/REPL/) by running `julia --project` and install the
+ClimaLand.jl package by running the following within the REPL:
 
 ```julia
-julia> using Pkg
-julia> Pkg.add(ClimaLand)
-julia> using ClimaLand
+using Pkg
+Pkg.add(ClimaLand)
 ```
 
-A typical land simulation employs several different parameterizations to model the various land-surface processes. Let's start our journey into ClimaLand by looking at one of those.
-
-### Parameterization
+## Running a simple soil model
 
-Let's start with a basic example: compute canopy gross photosynthesis (GPP).
+Now that we have Julia installed, let's set up and run a simple ClimaLand simulation!
+You can follow along by copying the following code segments into your REPL.
 
-```@repl
+First we import the necessary Julia packages:
+```julia
+import ClimaParams as CP
 using ClimaLand
-@doc ClimaLand.Canopy.compute_GPP
+using ClimaLand.Domains
+using ClimaLand.Soil
+import ClimaLand.Simulations: LandSimulation, solve!
+import ClimaLand.Parameters as LP
+using Dates
 ```
 
-As you can see, our parameterization for GPP is located in the [`Canopy`](@ref Photosynthesis) Module, and requires four arguments.
-For example, with An = 5 µmol m⁻² s⁻¹, K = 0.5, LAI = 3 m² m⁻², Ω = 0.7, you can compute GPP like below:
-
-```@repl
-import ClimaLand.Canopy as canopy
-canopy.compute_GPP(5.0, 0.5, 3.0, 0.7)
+Choose a floating point precision, and get the parameter set, which holds constants used across CliMA models.
+```julia
+FT = Float32
+earth_param_set = LP.LandParameters(FT);
 ```
 
-Et voilà!
-
-Note that our package [ParamViz](https://github.com/CliMA/ParamViz.jl) allows interactive visualisation of
-our parameterizations. See examples in the standalone models pages.
+We will run this simulation on a column domain with 1 meter depth, at a random lat/lon location.
+```julia
+zmax = FT(0)
+zmin = FT(-1.0)
+longlat = FT.((34.1, -118.1))
+domain = Domains.Column(; zlim = (zmin, zmax), nelements = 10, longlat);
+surface_space = domain.space.surface;
+```
 
-### ClimaLand structure
+We choose the initial and final simulation times as DatesTimes, and a timestep in seconds.
+```julia
+start_date = DateTime(2008);
+end_date = start_date + Second(60 * 60 * 72);
+dt = 1000.0;
+```
 
-ClimaLand contains multiple modules. They are listed below:
+The soil model takes in 2 forcing objects, atmosphere and radiation,
+which we read in from ERA5 data.
+```julia
+era5_ncdata_path =
```

Comment:
Maybe at the bottom we should cite this: Hersbach, Hans, et al. "The ERA5 global reanalysis." Quarterly journal of the royal meteorological society 146.730 (2020): 1999-2049.